src/main/java/Listmanager.java

```diff
@@ -0,0 +1,133 @@
+import java.util.Scanner;
+
+public class Todolist {
+
+    String Icon = "T";
+    private static final int MAX_LIST_LENGTH = 100;
+    static String[] todolist = new String[MAX_LIST_LENGTH];
+    static boolean[] isDoneList = new boolean[MAX_LIST_LENGTH];
+
+    static int ListCount= 0;
+    static Scanner in = new Scanner(System.in);
+
+    public static void AddToList(){
```

Comment:
```suggestion
    public static void AddToList() {
```

Take note that for function declaration, there should be a space before {